pkg/apis/postgres-operator.crunchydata.com/v1beta1/instrumentation_types.go

```diff
@@ -65,6 +65,15 @@ type InstrumentationConfigSpec struct {
 	// +listType=atomic
 	// +optional
 	Files []corev1.VolumeProjection `json:"files,omitempty"`
+
+	// EnvironmentVariables allows the user to add environment variables to the
+	// collector container.
+	// ---
+	// +kubebuilder:validation:MinItems=1
```

Comment:
What does this CEL rule do for an optional field? Is it, like, if you put this field in, it can't be blank? (Have you tested with clusters that have and don't have any custom env vars?)